recipes-core/images/nilrt-dkms-image.bb

```diff
@@ -1,5 +1,12 @@
 DESCRIPTION = "Runmode image for ethernet-based NI Linux Real-Time targets running XFCE (DKMS)."
 
+# Disable running ldconfig at the end of the rootfs creation process.
+# Running ldconfig removes symlinks created by ni packages installed 
+# to the rootfs, particularly the symlinks to libraries in 
+# NIWebServer. This needs to be investigated before ldconfig can be
+# reenabled.
+LDCONFIGDEPEND = ""
```

Comment:
Should this be a part of the `nilrt-internal.inc` instead? Is there a time that you want to include the proprietary content, but would like to keep `LDCONFIGDEPEND`?